canonical/ubuntu.com
# Setup redirects.json

Convert redirects.txt to our new format, add the new module.
# QA

``` bash
make clean  # Say "yes" to "destroy images"
docker rmi ubuntuwebsite_web
sudo rm .sass-cache
make run
```

and check that some of the redirects in `redirects.json` work.
